UnityProject/Assets/Scripts/Core/Addressables/TestAddressableScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Initialisation;
+using Newtonsoft.Json.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+
+public class TestAddressableScript : MonoBehaviour
+{
+
+	//[NaughtyAttributes.Button]
+	//[MenuItem("Tools/LoadCatalog")]
+	//public static void LoadCatalog()
```

Comment:
Everything potentially useful in this file is commented out. Could this file be deleted?

Reply:
will clean